k8s/namespaces/ccd/ccd-data-store-api/aat.yaml

```diff
@@ -16,9 +16,9 @@ spec:
       environment:
         CCD_CONDITIONAL_APIS_CASE_ASSIGNED_USER_AND_ROLES_ENABLED: true
         CCD_DOCUMENT_URL_PATTERN: ^https?://(((?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|(dm-store-aat\.service\.core-compute-aat|dm-store-(pr-[0-9]+|preview)\.service\.core-compute-preview)\.internal(?::\d+)?)\/documents\/[A-Za-z0-9-]+(?:\/binary)?)|((em-hrs-api-(?:aat|staging)\.service\.core-compute-aat|em-hrs-api-(pr-[0-9]+|preview)\.service\.core-compute-preview)\.internal(?::\d+)?\/hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9]+))
-        CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,divorce_frontend,civil_service,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,probate_backend
+        CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,divorce_frontend,civil_service,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,probate_backend,fis_hmc_api
```

Comment:
@marvincudjoe - Do we need this change?

Reply:
Not needed - I'll undo the change shortly